scripts/controllers/Lobby.gd

```diff
@@ -0,0 +1,29 @@
+extends Control
+
+
+func _ready():
+	var _junk = get_tree().connect("connected_to_server", self, "connected")
```

Comment:
I believe the server should be listening to
`network_peer_connected` and `network_peer_disconnected` to handle players joining and leaving.

`connected_to_server` is a client event, so currently the client is in charge of incrementing the players.